Samples/SwiftUITestbed/Sources/AppDelegate.swift

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MarketWorkflowUI
+import UIKit
+import WorkflowUI
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        window = UIWindow(frame: UIScreen.main.bounds)
+        window?.rootViewController = WorkflowHostingController(
+            workflow: RootWorkflow(close: nil)
+                .mapRendering(MarketRootScreen.init)
+                .mapRendering(ModalHostContainer.init)
```

Comment:
These need to be inverted:
```suggestion
                .mapRendering(ModalHostContainer.init)
                .mapRendering(MarketRootScreen.init)
```

The modal host will need to be contained within the ModalHost so that it can leverage the Market context information observed in a root for modal presentation styles.